<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>900</width>
                <height>600</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>900</width>
                <height>600</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>900</width>
                <height>600</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <widget class="QLabel" name="label_3">
                    <property name="font">
                        <font>
                            <pointsize>28</pointsize>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>File lock system</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="frame">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <spacer name="verticalSpacer_2">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <layout class="QFormLayout" name="formLayout">
                                <item row="0" column="0">
                                    <widget class="QLineEdit" name="TxtChooseFile">
                                        <property name="minimumSize">
                                            <size>
                                                <width>600</width>
                                                <height>25</height>
                                            </size>
                                        </property>
                                        <property name="placeholderText">
                                            <string>Choose your File Location</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="1">
                                    <widget class="QPushButton" name="BtnChooseFile">
                                        <property name="minimumSize">
                                            <size>
                                                <width>0</width>
                                                <height>27</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Select your file</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_3">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <layout class="QFormLayout" name="formLayout_2">
                                <item row="0" column="0">
                                    <widget class="QLabel" name="label">
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>14</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>Enter Your Password</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="1">
                                    <widget class="QLineEdit" name="TxtPassword">
                                        <property name="minimumSize">
                                            <size>
                                                <width>400</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>14</pointsize>
                                            </font>
                                        </property>
                                        <property name="echoMode">
                                            <enum>QLineEdit::PasswordEchoOnEdit</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="1">
                                    <spacer name="horizontalSpacer">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item row="3" column="0">
                                    <widget class="QLabel" name="label_2">
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>14</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>Confirm Your Password</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="3" column="1">
                                    <widget class="QLineEdit" name="TxtConfirmPassword">
                                        <property name="minimumSize">
                                            <size>
                                                <width>400</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>14</pointsize>
                                            </font>
                                        </property>
                                        <property name="echoMode">
                                            <enum>QLineEdit::PasswordEchoOnEdit</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="1">
                                    <widget class="QLabel" name="label_4">
                                        <property name="font">
                                            <font>
                                                <family>Rockwell Condensed</family>
                                                <pointsize>12</pointsize>
                                            </font>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">color: rgb(255, 0, 0);</string>
                                        </property>
                                        <property name="text">
                                            <string>Make Sure to rememeber your password and store it in safe place
                                            </string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_4">
                                <property name="font">
                                    <font>
                                        <family>Rockwell</family>
                                        <pointsize>14</pointsize>
                                    </font>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QPushButton" name="BtnSubmit">
                                        <property name="minimumSize">
                                            <size>
                                                <width>0</width>
                                                <height>25</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Submit</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="BtnExtract">
                                        <property name="minimumSize">
                                            <size>
                                                <width>0</width>
                                                <height>25</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Extract</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="BtnClose">
                                        <property name="minimumSize">
                                            <size>
                                                <width>0</width>
                                                <height>25</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Clear</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="BtnMainMenu">
                                        <property name="minimumSize">
                                            <size>
                                                <width>0</width>
                                                <height>25</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Close</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
